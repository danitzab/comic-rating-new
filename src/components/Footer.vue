<template>
  <footer class="font-montserrat px-6 py-5">
    <div class="has-text-centered">
      <div class="mb-2">Redes Sociales</div>
      <a
        class="mx-1"
        href="https://www.linkedin.com/in/danitzabonilla/"
        target="_blank"
        rel="noopener noreferrer"
      >
        <v-icon color="#ffff">mdi-linkedin</v-icon>
      </a>
      <a
        href="https://github.com/Danitzab"
        target="_blank"
        rel="noopener noreferrer"
        class="mx-1"
      >
        <v-icon color="#ffff">mdi-github</v-icon>
      </a>
      <v-divider></v-divider>
      <div class="text-center">Danitza Bonilla | Frontend Developer</div>
    </div>
  </footer>
</template>
