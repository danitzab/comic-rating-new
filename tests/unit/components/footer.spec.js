import Vue from 'vue'
import Vuetify from 'vuetify'
import { mount, createLocalVue } from '@vue/test-utils'
import Footer from '@/components/Footer.vue';

Vue.use(Vuetify)

const localVue = createLocalVue();

describe('module', () => {
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  it('should do something', () => {
    const wrapper = mount(Footer, {
      localVue,
      vuetify,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
