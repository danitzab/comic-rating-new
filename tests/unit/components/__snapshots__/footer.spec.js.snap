// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`module should do something 1`] = `
<footer class="font-montserrat px-6 py-5">
  <div class="has-text-centered">
    <div class="mb-2">Redes Sociales</div> <a href="https://www.linkedin.com/in/danitzabonilla/" target="_blank" rel="noopener noreferrer" class="mx-1"><i aria-hidden="true" class="v-icon notranslate mdi mdi-linkedin theme--light" style="color: rgba(255, 255, 255, 1); caret-color: #ffff;"></i></a> <a href="https://github.com/Danitzab" target="_blank" rel="noopener noreferrer" class="mx-1"><i aria-hidden="true" class="v-icon notranslate mdi mdi-github theme--light" style="color: rgba(255, 255, 255, 1); caret-color: #ffff;"></i></a>
    <hr role="separator" aria-orientation="horizontal" class="v-divider theme--light">
    <div class="text-center">Danitza Bonilla | Frontend Developer</div>
  </div>
</footer>
`;
